app/kumactl/cmd/install/testdata/install-transparent-proxy.dns.golden.txt

```diff
@@ -1,19 +1,21 @@
-\* nat
--N (.*)_INBOUND
--N (.*)_OUTBOUND
--N (.*)_INBOUND_REDIRECT
--N (.*)_OUTBOUND_REDIRECT
--A PREROUTING -p tcp -j (.*)_INBOUND
+* nat
+-N KUMA_MESH_INBOUND
+-N KUMA_MESH_OUTBOUND
+-N KUMA_MESH_INBOUND_REDIRECT
+-N KUMA_MESH_OUTBOUND_REDIRECT
+-A PREROUTING -p tcp -j KUMA_MESH_INBOUND
 -I OUTPUT 1 -p udp --dport 53 -m owner --uid-owner 0 -j DOCKER_OUTPUT
 -I OUTPUT 2 -p udp --dport 53 -j REDIRECT --to-ports 12345
--A OUTPUT -p tcp -j (.*)_OUTBOUND
--A (.*)_INBOUND -p tcp -j (.*)_INBOUND_REDIRECT
--A (.*)_OUTBOUND -s (.*) -o (.*) -j RETURN
--A (.*)_OUTBOUND -p tcp ! --dport 53 -o (.*) ! -d (.*) -m owner --uid-owner 0 -j (.*)_INBOUND_REDIRECT
--A (.*)_OUTBOUND -p tcp ! --dport 53 -o (.*) -m owner ! --uid-owner 0 -j RETURN
--A (.*)_OUTBOUND -m owner --uid-owner 0 -j RETURN
--A (.*)_OUTBOUND -p tcp --dport 53 -j REDIRECT --to-ports 12345
--A (.*)_OUTBOUND -d (.*) -j RETURN
--A (.*)_OUTBOUND -j (.*)_OUTBOUND_REDIRECT
--A (.*)_INBOUND_REDIRECT -p tcp -j REDIRECT --to-ports (15006|15010)
--A (.*)_OUTBOUND_REDIRECT -p tcp -j REDIRECT --to-ports 15001
+-A OUTPUT -p tcp -j KUMA_MESH_OUTBOUND
+-A KUMA_MESH_INBOUND -p tcp -j KUMA_MESH_INBOUND_REDIRECT
+-A KUMA_MESH_OUTBOUND -s subnetPlaceholder/32 -o ifPlaceholder -j RETURN
+-A KUMA_MESH_OUTBOUND -p tcp ! --dport 53 -o ifPlaceholder ! -d subnetPlaceholder/32 -m owner --uid-owner 0 -j KUMA_MESH_INBOUND_REDIRECT
+-A KUMA_MESH_OUTBOUND -p tcp ! --dport 53 -o ifPlaceholder -m owner ! --uid-owner 0 -j RETURN
+-A KUMA_MESH_OUTBOUND -m owner --uid-owner 0 -j RETURN
+-A KUMA_MESH_OUTBOUND -p tcp --dport 53 -j REDIRECT --to-ports 12345
+-A KUMA_MESH_OUTBOUND -d subnetPlaceholder/32 -j RETURN
+-A KUMA_MESH_OUTBOUND -j KUMA_MESH_OUTBOUND_REDIRECT
+-A KUMA_MESH_INBOUND_REDIRECT -p tcp -j REDIRECT --to-ports 15006
```

Comment:
Something is odd here I think

Reply:
15006 vs 15010 I'm not sure why this used to be an option

Reply:
Gotcha thx

Reply:
I have checked and there will be a problem if tests will be run on a machine with IPv6.
```sh
 ~/kuma (master)> go run app/kumactl/main.go install transparent-proxy --dry-run --kuma-dp-user kuma-dp
* nat
-N KUMA_MESH_INBOUND
-N KUMA_MESH_OUTBOUND
-N KUMA_MESH_INBOUND_REDIRECT
-N KUMA_MESH_OUTBOUND_REDIRECT
-A PREROUTING -p tcp -j KUMA_MESH_INBOUND
-A OUTPUT -p tcp -j KUMA_MESH_OUTBOUND
-A KUMA_MESH_INBOUND -p tcp -j KUMA_MESH_INBOUND_REDIRECT
-A KUMA_MESH_OUTBOUND -s ::6/128 -o lo -j RETURN
-A KUMA_MESH_OUTBOUND -p tcp -o lo ! -d ::1/128 -m owner --uid-owner 5678 -j KUMA_MESH_INBOUND_REDIRECT
-A KUMA_MESH_OUTBOUND -p tcp -o lo -m owner ! --uid-owner 5678 -j RETURN
-A KUMA_MESH_OUTBOUND -m owner --uid-owner 5678 -j RETURN
-A KUMA_MESH_OUTBOUND -d ::1/128 -j RETURN
-A KUMA_MESH_OUTBOUND -j KUMA_MESH_OUTBOUND_REDIRECT
-A KUMA_MESH_INBOUND_REDIRECT -p tcp -j REDIRECT --to-ports 15010
-A KUMA_MESH_OUTBOUND_REDIRECT -p tcp -j REDIRECT --to-ports 15001
COMMIT
Transparent proxy set up successfully, you can now run kuma-dp using transparent-proxy.
```